bundledata.go

```diff
@@ -455,6 +459,41 @@ func (verifier *bundleDataVerifier) verifyRelations() {
 	}
 }
 
+func (verifier *bundleDataVerifier) verifyEndpointBindings() {
+	for name, svc := range verifier.bd.Services {
+		charm, ok := verifier.charms[name]
+		// Only thest the ok path here because the !ok path is tested in verifyServices
+		if ok {
+			for BoundInterface := range svc.EndpointBindings {
```

Comment:
I think this needs to be called "endpoint", as it's the relation name we're binding to a space, not the interface of the relation. E.g. with a metadata like this:
```yaml
provides:
    db:
      interface: mysql
```
and bundle.yaml having:
```yaml
...
    bindings:
        db: storage
...
```
"db" (not "mysql" - the interface) is bound to space "storage".